DistributedCollective/Sovryn-frontend
# feat: add rdoc to portfolio and allow rdoc conversion to xusd

- Have new token RDOC in portfolio
- Allow RDOC to be converted to XUSD (like RUSDT)

testing:
1. verify RDOC token is in portfolio page.
2. verify RDOC token has button "Convert to XUSD".
3. click "Convert to XUSD" and try to convert it.
4. verify you receive 1 XUSD for each RDOC you converted in "Convert to XUSD" modal.
